app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -59,10 +61,16 @@ public void onItemUserNameClick(TimelineModel item) {
                     mContext.startActivity(
                             ProfileUserActivity.getInstance(mContext, item.getCreatedUser()));
                 }
+
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.